tokens/src/commands.rs

```diff
@@ -396,6 +410,7 @@ fn send_messages(
             } else {
                 let (blockhash, last_valid_block_height) =
                     client.get_latest_blockhash_with_commitment(CommitmentConfig::default())?;
+                message.recent_blockhash = blockhash;
```

Comment:
Is this necessary? The transaction signing logic should automatically replace the blockhash in the message

Reply:
Good catch, I have removed this line

Reply:
This is not necessary, and has been removed.